src/sidenav-expanded.styl

```diff
@@ -3,7 +3,9 @@
 
     .sidenav-nav > .sidenav-navitem {
         .navicon + .navtext {
-            display: inline-block;
+            visibility: visible
```

Comment:
Although it doesn't matter, it's better to keep consistency of preserving trailing semicolons. 🙂